src/index.ts

```diff
@@ -1,23 +1,28 @@
-import users from './users';
 import meetings from './meetings';
-import webinars from './webinars';
+import recordings from './recordings';
 import reports from './reports';
+import users from './users';
+import webinars from './webinars';
+
 import { ZoomOptions } from './common';
 import request from './util/request';
 
 export default function(zoomApiOpts: ZoomOptions) {
   const zoomRequest = request(zoomApiOpts);
 
   return {
-    users: users(zoomRequest),
     meetings: meetings(zoomRequest),
-    webinars: webinars(zoomRequest),
+    recordings: recordings(zoomRequest),
     reports: reports(zoomRequest),
+    users: users(zoomRequest),
+    webinars: webinars(zoomRequest),
   };
 }
 
-export * from './webhooks';
 export * from './common';
-export * from './users';
 export * from './meetings';
+export * from './recordings';
```

Comment:
Was the reordering just to make the exported filenames alphabetical? If so that's completely fine.

Reply:
Yes!